client/components/jetpack/sidebar/index.jsx

```diff
@@ -1,83 +0,0 @@
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import Sidebar from 'calypso/layout/sidebar';
-import SidebarFooter from 'calypso/layout/sidebar/footer';
-import SidebarItem from 'calypso/layout/sidebar/item';
-import SidebarMenu from 'calypso/layout/sidebar/menu';
-import SidebarRegion from 'calypso/layout/sidebar/region';
-import CurrentSite from 'calypso/my-sites/current-site';
```

Comment:
There's some more cleanup opportunity here - this seems to be the last usage of this component.